tests/test_core.py

```diff
@@ -7282,10 +7203,9 @@ def test_emscripten_log(self):
     self.emcc_args += ['-DRUN_FROM_JS_SHELL']
     self.do_run_in_out_file_test('tests', 'emscripten_log', 'emscripten_log.cpp')
     # test closure compiler as well
-    if self.run_name == 'asm2':
-      print('closure')
-      self.emcc_args += ['--closure', '1', '-g1'] # extra testing
-      self.do_run_in_out_file_test('tests', 'emscripten_log', 'emscripten_log_with_closure')
+    print('closure')
```

Comment:
how about calling `maybe_closure()`, so it doesn't slow down all modes?

Reply:
maybe_closure always returns False when using `-g` and this test always sets `-g`

Reply:
Oh, I see, the `-g1` disables the earlier `-g`...

Oh well, this might be ok, but closure is slower, and I'm not sure this extra testing is worth it. Do you have a sense of why closure might matter here specifically?

Reply:
I snuck this into #12471 where it made mode sense.   Its no longer part of this change anyway.